### PR TITLE
Add command for SCSS

### PR DESCRIPTION
**Pull request by aNickzz**
Thought I should PR this one in, I've been using it for a while.

### CODE REVIEW

**Comment by aNickzz** [comment]
I didn't put it in my PR but its probably worth sorting that list alphabetically. At the very least it should help avoid conflicts when there's multiple PR's adding items to it.

File: package.json [hunk 1]
```diff
@@ -161,7 +161,8 @@
             "nim": "nim compile --verbosity:0 --hints:off --run",
             "lisp": "sbcl --script",
             "kit": "kitc --run",
-            "v": "v run"
+            "v": "v run",
+            "scss": "scss --style expanded"
```
Comment (formulahendry) [inline package.json]: Is it scss or sass? https://sass-lang.com/install
Sorry, I am not quite familiar with this. Just want to have a double confirm. :)

Comment (aNickzz) [reply package.json]: Both exist :)

```bash
sorokin@here:~ $ sass -v
Sass 3.4.21 (Selective Steve)
sorokin@here:~ $ scss -v
Sass 3.4.21 (Selective Steve)
```

Comment (formulahendry) [reply package.json]: Thanks for double confirm. :)
BTW, is `"scss": "scss"` enough? It seems that `--style expanded` is already the default parameter.

Comment (aNickzz) [reply package.json]: Maybe it depends on which scss compiler you have installed.

`scss`
```css
.test {
  foo: bar; }
```

`scss --style expanded`
```css
.test {
  foo: bar;
}
```

In any case, I'd prefer to be explicit, we're not typing in the full command manually so there's no penalty for writing it out even if it was the default.

Comment (aNickzz) [reply package.json]: Re: scss vs sass
https://stackoverflow.com/a/22069780/6398754
https://stackoverflow.com/a/31403160/6398754

Comment (formulahendry) [reply package.json]: Awesome! Thanks for the detailed explanation!

**Pull request merged**